minim-openwifi-scripts/files/sbin/leds.sh

```diff
@@ -467,10 +467,10 @@ led_firmware_update() {
             echo ZW_CODE_05 > /dev/ttyMSM1
         elif [ "$(board_name)" == "motorola,r14" ] ; then
             _none_trigger "$AMBER"
-            _brightness_zero "$AMBER"
-            _timer_trigger "$GREEN"
-            _rapidly_blink "$GREEN"
-            _brightness_full "$GREEN"
+            _brightness_full "$AMBER"
```

Comment:
Aren't the amber led and power led supposed to be blinking simultaneously in this situation?  I expected to see something like:
```
_shift_while _timer_trigger "$POWER $AMBER"
_rapidly_blink "$POWER $AMBER"
_brightness_full "$POWER $AMBER"
```

Reply:
Working on it.

Reply:
I though the amber led should blink static.